gobblin-compaction/src/main/java/gobblin/compaction/mapreduce/MRCompactorJobRunner.java

```diff
@@ -129,23 +135,23 @@ protected MRCompactorJobRunner(Dataset dataset, FileSystem fs, Double priority)
     this.outputPath = new Path(jobProps.getProp(MRCompactor.COMPACTION_JOB_DEST_DIR));
     this.tmpPath = new Path(jobProps.getProp(MRCompactor.COMPACTION_JOB_TMP_DIR));
     this.fs = fs;
-    this.perm = HadoopUtils.deserializeFsPermission(this.jobProps, COMPACTION_JOB_OUTPUT_DIR_PERMISSION,
-        FsPermission.getDefault());
+    this.perm =
+        HadoopUtils.deserializeFsPermission(this.jobProps, COMPACTION_JOB_OUTPUT_DIR_PERMISSION,
+            FsPermission.getDefault());
     this.deduplicate =
         this.jobProps.getPropAsBoolean(MRCompactor.COMPACTION_DEDUPLICATE, MRCompactor.DEFAULT_COMPACTION_DEDUPLICATE);
     this.priority = priority;
-    this.eventSubmitter = new EventSubmitter.Builder(
-        GobblinMetrics.get(this.jobProps.getProp(ConfigurationKeys.JOB_NAME_KEY)).getMetricContext(),
-        MRCompactor.COMPACTION_TRACKING_EVENTS_NAMESPACE).build();
+    this.eventSubmitter =
+        new EventSubmitter.Builder(GobblinMetrics.get(this.jobProps.getProp(ConfigurationKeys.JOB_NAME_KEY))
+            .getMetricContext(), MRCompactor.COMPACTION_TRACKING_EVENTS_NAMESPACE).build();
   }
 
   @Override
   public void run() {
-
     Configuration conf = HadoopUtils.getConfFromState(this.jobProps);
-    DateTime jobStartTime = new DateTime(DateTimeZone
-        .forID(this.jobProps.getProp(MRCompactor.COMPACTION_TIMEZONE, MRCompactor.DEFAULT_COMPACTION_TIMEZONE)));
-
+    DateTime jobStartTime =
+        new DateTime(DateTimeZone.forID(this.jobProps.getProp(MRCompactor.COMPACTION_TIMEZONE,
+            MRCompactor.DEFAULT_COMPACTION_TIMEZONE)));
     try {
       if (this.jobProps.getPropAsBoolean(MRCompactor.COMPACTION_JOB_LATE_DATA_MOVEMENT_TASK, false)) {
         List<Path> lateFilePaths = Lists.newArrayList();
```

Comment:
Please rename to `newLateFilePaths` to avoid confusion with files in the `daily_late` folder.